### DISCUSSION TITLE
【通知】开往域名换为 travellings.cn

### DISCUSSION THREAD

**Discussion by linlinzzo** [travellings-link/travellings#1748] [公告]
各位开往成员：
travellings.cn 的域名ICP备案已通过，请各位开往成员将开往的链接换为 https://www.travellings.cn/go.html ，谢谢。

现在，开往在跳转到成员网站时会添加一个`?utm_source=travellings`，网站可以用这个统计开往给网站带来的流量。（该修改处于试验阶段）
目前，有一些彩蛋······已经布置好了······

**Comment by wangzi0218** [comment]
感谢体系， [见字如面](https://hiwannz.com/) 已经完成了更新。

**Comment by ChrisKimZHT** [comment]
加上的参数会导致有些博客 404，估计是配置文件有问题，建议提醒一下站长。

**Comment by linlinzzo** [comment]
> 加上的参数会导致有些博客 404，估计是配置文件有问题，建议提醒一下站长。

具体可以说下是哪些404了吗？

**Comment by ChrisKimZHT** [comment]
> 具体可以说下是哪些404了吗？

我现在还只碰到一个：https://kuaikan.ink/?utm_source=travellings

**Comment by linlinzzo** [comment]
> 我现在还只碰到一个：https://kuaikan.ink/?utm_source=travellings

这个参数年后删掉吧。

**Comment by wangzi0218** [comment]
<img width="769" alt="image" src="https://user-images.githubusercontent.com/7765857/221482677-afe80295-5bdc-46a1-9e6f-4a20886937f0.png">

**Comment by ChrisKimZHT** [comment]
确实是炸了

![image](https://user-images.githubusercontent.com/49368462/221507211-b3d67389-9674-4099-8d56-196dd6c1dd06.png)

**Comment by Tseing** [comment]
README.md 包括 issue 的正文中写的是 `https://www.travellings.cn/go.html`，这个 URL 配置得有问题，用 curl 检查结果如下

```
curl -L https://www.travellings.cn
curl: (47) Maximum (50) redirects followed
```
所以就会有 @wangzi0218 君出现的情况。
`https://travellings.cn/go.html` 是目前正常的 URL，我看了一圈，链接基本都挂了 0_0

**Comment by wangzi0218** [comment]
> README.md 包括 issue 的正文中写的是 `https://www.travellings.cn/go.html`，这个 URL 配置得有问题，用 curl 检查结果如下
> 
> ```
> curl -L https://www.travellings.cn
> curl: (47) Maximum (50) redirects followed
> ```
> 
> 所以就会有 @wangzi0218 君出现的情况。 `https://travellings.cn/go.html` 是目前正常的 URL，我看了一圈，链接基本都挂了 0_0

感谢提示，我修改了一下自己博客里面的链接，顺便发现 [www.trav](https://www.travellings.cn/assests/w.png)资源也都挂掉了。

**Comment by raingnight** [comment]
https://www.travellings.cn 挂了，不加 www 可正常访问

**Comment by Big-Cake-jpg** [comment]
www.travellings.cn 无法访问是已知问题。

由于主页的服务器相关事宜由 @linlinzzo 负责且其学业繁忙，该问题可能将延迟到周末方能进行处理。

在此之前请先通过去除开往外链中的 www 来缓解。

**Comment by linlinzzo** [comment]
@wangzi0218 @Tseing @ChrisKimZHT @raingnight  无法访问的问题已解决，感谢反馈。如果在逛开往的时候发现失效网站，请及时跟我们反馈。

**Comment by arect** [comment]
证书过期了。
![image](https://user-images.githubusercontent.com/40746088/230555128-3496e524-6965-466b-83d8-fcd9fc997cc2.png)
